### DISCUSSION TITLE
Activate monitoring / routing in background with automation?

### DISCUSSION THREAD

**Discussion by hansonxyz** [Mygod/VPNHotspot#522] [Q&A]
Hello,

Is there a way to enable routing wifi traffic through the VPN through ap_br_wlan2 without opening the application?  I currently am using tasker and I can use tasker to start the hotspot, start wireguard, but I can't figure out how to start VPNHotspot and have it setup traffic routing without unlocking the phone and starting the application in foreground.  

I was able to modify the source code so as soon as the application was opened, the monitor option was picked on my ap_br_wlan2 connection.  So I can kind of make this work by using tasker to open the modified application, let it be in foreground for a few moments, then quit the application.  But I never figured out how I could modify the source code to start the routing without opening the application.

Is there a way to do this, that I am not aware of?

If not, could this be a feature request?

Thank you,

**Comment by Mygod** [comment]
Enable monitor in top right.
